static/js/protein/chart.ts

```diff
@@ -905,6 +907,9 @@ export function drawDiseaseGeneAssocChart(
     .attr("width", (d) => x(d.value))
     .attr("height", y.bandwidth())
     .style("fill", BAR_COLOR)
+    .on("click", function (d) {
+      window.open(GRAPH_BROWSER_REDIRECT + d.id);
```

Comment:
did you want the link to open in the same tab or different tab? I think this will open the url in a new tab. To open in current tab, you want `window.open(GRAPH_BROWSER_REDIRECT + d.id, "_self")`

Reply:
I think it will be better to have it in a new tab!